src/test/ui/issues/issue-104870.rs

```diff
@@ -0,0 +1,19 @@
+// Avoid panicking if the Clone trait is not found while building error suggestions
```

Comment:
Please move this test and give it a descriptive name. Thanks for catching this @Nilstrieb.

Reply:
Done.